src/Symfony/Component/Intl/Languages.php

```diff
@@ -98,7 +98,12 @@ public static function getAlpha2Code(string $language): string
      */
     public static function getAlpha3Codes(): array
     {
-        return self::readEntry(['Alpha2ToAlpha3'], 'meta');
+        return self::asort(array_unique(array_merge(
```

Comment:
A lot of dynamic transformation going on here.

I hope you plan to move this logic to the generator, so we can have efficient getters in the `Languages` class.

Reply:
i guess that's the better approach yes, but the data will increase significantly (though it may still be the preferred approach :))

So yes, i think we should introduce a separate "alpha3" code list, apart from the current mappings. This means `getAlpha3Codes` will be a plain indexed array (numeric keys), like `getLanguageCodes()`. I tend to follow the same logic for Countries also.

Reply:
Done.